modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp

```diff
@@ -76,6 +81,12 @@
             .setResponseType(authzGrantType)
             .setScope(scope);
 
+    if (formPostMode) {
+        oAuthPKCEAuthenticationRequestBuilder.setParameter(OAuth2Constants.OAUTH2_RESPONSE_MODE, OAuth2Constants.OAUTH2_FORM_POST);
+    }
+
```

Comment:
It's better to remove unnecessary new lines.

Reply:
Fix with https://github.com/wso2/product-is/pull/1369